includes/class-newspack-popups.php

```diff
@@ -76,8 +76,8 @@ public function __construct() {
 			add_action( 'customize_controls_enqueue_scripts', [ __CLASS__, 'enqueue_customizer_assets' ] );
 			add_filter( 'display_post_states', [ __CLASS__, 'display_post_states' ], 10, 2 );
 			add_action( 'save_post_' . self::NEWSPACK_POPUPS_CPT, [ __CLASS__, 'popup_default_fields' ], 10, 3 );
-			add_action( 'transition_post_status', [ __CLASS__, 'remove_default_category' ], 10, 3 );
-
+			add_action( 'transition_post_status', [ __CLASS__, 'prevent_default_category_on_publish' ], 10, 3 );
+			add_filter( 'pre_delete_term', [ __CLASS__, 'prevent_default_category_on_term_delete' ], 10, 2 );
```

Comment:
[`pre_delete_term`](https://developer.wordpress.org/reference/hooks/pre_delete_term/) is an action hook

Reply:
Ack, thanks! Updated in 692ac31. I was also wondering why PHPCS was telling me that callback needed a return value. 😆 I removed those in df0ddef as it doesn't need to return anything.